tests/jvm/src/test/scala/scala/meta/tests/semanticdb/ExpectSuite.scala

```diff
@@ -48,6 +48,14 @@ class ExpectSuite extends FunSuite with DiffAssertions {
         import MetacIndexExpect._
         assertNoDiff(loadObtained, loadExpected)
       }
+      test("javac.expect") {
+        import JavacExpect._
+        assertNoDiff(loadObtained, loadExpected)
+      }
+      test("javac.index") {
+        import JavacIndexExpect._
+        assertNoDiff(loadObtained, loadExpected)
+      }
```

Comment:
This conflicts with https://github.com/scalameta/scalameta/pull/1682. I'll let @olafurpg decide in which order we want to merge the PRs.